webknossos/webknossos/_nml/parameters.py

```diff
@@ -3,7 +3,11 @@
 
 from loxun import XmlWriter
 
-from .utils import IntVector6, Vector3, enforce_not_null, filter_none_values
+from webknossos.geometry import BoundingBox
+
+from .utils import Vector3, enforce_not_null, filter_none_values
+
+DEFAULT_COLOR = [0.2, 0.5, 0.1, 1]
```

Comment:
Where does this come from? Maybe rather `DEFAULT_BOUNDING_BOX_COLOR`?

Reply:
I renamed it now. I just chose some arbitrary color. There is no specific reason for it being the way it is.